index.js

```diff
@@ -1,8 +1,7 @@
 'use strict';
 const os = require('os');
-const toarray = require('lodash/toArray');
-const defaults = require('lodash/defaults');
-const map = require('lodash/map');
+const defaults = require('lodash.defaults');
+const map = require('lodash.map');
```

Comment:
We can't use native `map` because Object.values are missing from Node.js 6.0.0. Works in latest Node.js 6.x, but better follow semver here.